ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeer.java

```diff
@@ -106,13 +127,45 @@ public EthPeer(
     peerValidators.forEach(peerValidator -> validationStatus.put(peerValidator, false));
     fullyValidated.set(peerValidators.isEmpty());
 
+    this.requestManagers = new HashMap<>();
```

Comment:
Thanks for applying the static map creation below, the same could be applied here, if the init methods return the map values.

Reply:
not really in a next PR this init will depend on flags

Reply:
ok